prosemirror-jdita/src/schema.ts

```diff
@@ -40,6 +63,11 @@ export const SCHEMAS: Record<string, (node: typeof BaseNode, next: (nodeName: st
     return result;
   },
 }
+
+/**
+ * `SCHEMA_CONTENT` is the LwDita schema
```

Comment:
`SCHEMA_CONTENT is the LwDita schema it's being used to get nodes allowed children` -> `The LwDITA Schema. Describes parent-child relationships.`